app/pages/reporter/ciip-application.tsx

```diff
@@ -41,8 +50,23 @@ class CiipApplication extends Component<Props> {
   });
 
   render() {
-    const {query} = this.props;
+    const {query, router} = this.props;
     const {session} = query || {};
+    const {application} = query || {};
+
+    if (
+      application &&
+      application.latestDraftRevision?.legalDisclaimerAccepted === false
```

Comment:
optional chaining removes the need for the `application && ...` check ```suggestion
      application?.latestDraftRevision?.legalDisclaimerAccepted === false
```